README.md

```diff
@@ -242,6 +250,60 @@ docker run -p 8080:80 \
    --num-shard 8
 ```
 
+### llava-v1.6-mistral-7b-hf BF16 on 1 Gaudi2 card
+
+Image usually occupy about 2000 tokens in input, For examples image of size 512x512 occupys about 2800 tokens. max-input-tokens must be larger than the token number of image. Otherwise the image may be truncated. We set the BASE_IMAGE_TOKENS=2048 as the default image tokens number, it is the minimum value of max-input-tokens. If you want to change the BASE_IMAGE_TOKENS value,
+Please set the environment variable BASE_IMAGE_TOKENS. The warmup will generate graphs with input length from BASE_IMAGE_TOKENS to max-input-tokens. For LLava-next 7B model the maximum value of max-batch-prefill-tokens is 16384. The max-batch-prefill-tokens is an integer multiple of max-input-tokens. because of prefill_batch_size = max-batch-prefill-tokens/max-input-tokens.
+
+```bash
+model=llava-hf/llava-v1.6-mistral-7b-hf
+hf_token=YOUR_ACCESS_TOKEN   # HF access token
+volume=$PWD/data   # share a volume with the Docker container to avoid downloading weights every run
+
+docker run -p 8080:80 \
+   --runtime=habana \
+   -v $volume:/data \
+   -e HABANA_VISIBLE_DEVICES=all \
+   -e OMPI_MCA_btl_vader_single_copy_mechanism=none \
+   -e HF_HUB_ENABLE_HF_TRANSFER=1 \
+   -e HUGGING_FACE_HUB_TOKEN=$hf_token \
+   --cap-add=sys_nice \
+   --ipc=host \
+   ghcr.io/huggingface/tgi-gaudi:2.0.1 \
+   --model-id $model \
+   --max-input-tokens 4096 \
+   --max-batch-prefill-tokens 16384 \
+   --max-total-tokens 8192
+```
+
+Please implement the inference client according to the [tutorial](https://github.com/huggingface/tgi-gaudi/blob/habana-main/docs/source/basic_tutorials/visual_language_models.md).
```

Comment:
I also tried using curl, and it works for me, can we keep it simple 
`
curl -N 127.0.0.1:8010/generate_stream    -X POST     \
-d '{"inputs":"![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/rabbit.png)What is this a picture of?\n\n","parameters":{"seed": 42}}'    \
 -H 'Content-Type: application/json'
`

Reply:
Done